docs/source/markdown/podman-rm.1.md

```diff
@@ -22,6 +22,10 @@ Remove all containers.  Can be used in conjunction with -f as well.
 
 Read container ID from the specified file and remove the container.  Can be specified multiple times.
 
+**--ignore-no-such-container**
```

Comment:
I'm with mheon, I don't like the name, but can't think of better.  If you think it's unlikely we'll use other options starting with 'i' in the future, perhaps we could also accept `-i` as a short form of this option?

Reply:
`--allow-missing` maybe?

Reply:
`--allow-missing` it will be